lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -500,8 +514,8 @@ putSignedTx payload inputs outputs witnesses = do
     putWitness :: TxWitness -> Put
     putWitness (TxWitness bytes) = putByteString bytes
 
-putTx :: [(TxIn, Coin)] -> [TxOut] -> Put
-putTx inputs outputs = do
+putTx :: Put -> [(TxIn, Coin)] -> [TxOut] -> Put
+putTx putPayload inputs outputs = do
```

Comment:
`putTx` is called by both `signData` and `putSignedTx` witch both should `putPayload`.